### PR TITLE
feat: header footer styling

### PR DESCRIPTION
**Pull request by BenjaminYong**
## Purpose

Add styling to the footer tabs and stack headers for the app.

## Changes

- tab styling according to wireframes
- header styling for each stack (couldn't figure a way to make this a custom component, lemme know otherwise)
- icon change for SDGs

### Screenshots

<div>
<image src='https://scontent-syd2-1.xx.fbcdn.net/v/t1.15752-9/s1080x2048/121673328_382961309404260_7925776059440705265_n.jpg?_nc_cat=103&_nc_sid=ae9488&_nc_ohc=msK47FX_RMUAX9rPcJW&_nc_ht=scontent-syd2-1.xx&tp=7&oh=e717815c33b521e7cb8488e2d9c657d6&oe=5FAF597C' width='25%' height='25%'/>
<image src='https://scontent-syd2-1.xx.fbcdn.net/v/t1.15752-9/s1080x2048/121528223_348699699525974_5437086590038980898_n.jpg?_nc_cat=109&_nc_sid=ae9488&_nc_ohc=AxdgVSh_00sAX9exghZ&_nc_ht=scontent-syd2-1.xx&tp=7&oh=b3a2efb5506118c3a72bb46d587957f3&oe=5FB17051' width='25%' height='25%'/>
<image src='https://scontent-syd2-1.xx.fbcdn.net/v/t1.15752-9/s1080x2048/121669052_2805972386394492_3809138962726402417_n.jpg?_nc_cat=100&_nc_sid=ae9488&_nc_ohc=9XGHQGisdqcAX8DBid1&_nc_ht=scontent-syd2-1.xx&tp=7&oh=1b20967d347e44e8e4f0db6ad89de0b5&oe=5FB007AC' width='25%' height='25%'/>
<image src='https://scontent-syd2-1.xx.fbcdn.net/v/t1.15752-9/s1080x2048/121800222_997784627382534_5665490398747168421_n.jpg?_nc_cat=105&_nc_sid=ae9488&_nc_ohc=SkuBDELGpc8AX91L1D8&_nc_ht=scontent-syd2-1.xx&tp=7&oh=4cb4944876862929a6e9adde308ea872&oe=5FB17DB7' width='25%' height='25%'/>
</div>

## Checklist

- [x] Tested on Android
- [ ] Tested on iOS

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -51,7 +52,17 @@ function App() {
           {user ? (
             <MainScreen />
           ) : (
-            <Stack.Navigator>
+            <Stack.Navigator screenOptions={{
```
Comment (jonowu) [inline App.js:55]: I think that screenOptions could be an object in constants that you import and reuse

**Review by jonowu — CHANGES_REQUESTED** [review]

**Comment by jonowu** [comment]
Nice, it's all coming together. Just that one change and we should be set

File: App.js (hunk 1 above)
Comment (jonowu) [reply App.js:55]: The file could probably be called `screenOptions.js`

**Comment by BenjaminYong** [comment]
> Nice, it's all coming together. Just that one change and we should be set

Thank u sir, made those screen options into a constant

**Review by jonowu — APPROVED** [review]
LGTM

**Pull request merged**